helm/values.yaml

```diff
@@ -83,4 +83,50 @@ prometheus:
       memory: 100Mi
     limits:
       cpu: 500m
-      memory: 2Gi
\ No newline at end of file
+      memory: 2Gi
+
+elasticsearch:
+  namespace: efk
+  replicas: 3
+  env:
+    javaOpts: "-Xms512m -Xmx512m"
+    maxMapCount: 262144
+    ulimit: 65536
+  resources:
+    limits:
+      cpu: 1000m
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 256Mi
+  volumeClaim:
+    accessMode: ReadWriteOnce
```

Comment:
If the K8s cluster have multiple nodes, should we use ReadWriteOnce or ReadWriteMany?